plugins/janus_videoroom.c

```diff
@@ -6742,7 +6753,7 @@ static gboolean janus_gst_create_pipeline(forward_media_type media_type,
 	GError *error = NULL;
 	janus_gstr *gstr = NULL;
 	char rtsp_full_url[JANUS_RTP_FORWARD_STRING_SIZE] = {0};
-
+	char rtsp_url_base[JANUS_RTP_FORWARD_STRING_SIZE] = {0};
```

Comment:
I am using it ,I take the base to build the rtsp url for sgw and the full url inculdes the room id and media type,
previously instead of the base url you had the url from the config